docs/ubuntu-24.md

```diff
@@ -0,0 +1,61 @@
+## Installing on Ubuntu 24.04
+
+Ubuntu 24.04 [introduced restrictions](https://discourse.ubuntu.com/t/ubuntu-24-04-lts-noble-numbat-release-notes/39890) for unpriviledged user namespaces that prevents electron based apps from running without an appropriate apparmor profile.
+
+## Solution 1
+
+Install launcher using the .deb file. This will create an appropriate apparmor profile under the hood for you and you don't need to bother about it.
+
+## Solution 2
+
+If you don't want to use the .deb file but the AppImage, then you will need to create an apparmor profile yourself that points to the location of the AppImage. Follow the instructions below, replacing the launcher version wherever necessary with the actual launcher version that you're trying to use.
+
+1. Install tools
+`sudo apt install apparmor-easyprof apparmor-notify apparmor-utils`
```

Comment:
Is the installation of these tools strictly necessary? From the steps that I understood from you @pdaoust the definition of the app being unconfined can also just be added to the apparmor profile file itself without running any commands. Or did you have to run commands as well?

Reply:
I don't recall having to install any new packages to make this work; I just copied some random profile from another Electron-based app that ships in the Ubuntu repos.

Reply:
Okay, I just checked and I have none of those installed -- and in fact `apparmor-easyprof` doesn't even exist in the Ubuntu repos.